src/main/webapp/scripts/script.js

```diff
@@ -11,6 +11,19 @@
 //  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
+
+
+function count(input_id, counter_id, length) {
+    let el = document.getElementById(counter_id);
+    el.innerHTML = "<text>" + document.getElementById(input_id).value.length + length + "</text>";
+}
+
+let textInput = document.getElementById("input-text");
```

Comment:
I would avoid using global variables. I explained some of the reasons here https://github.com/googleinterns/step230-2020/pull/45 (first comment).